### PR TITLE
Filter Spark event logger outside the spark engine

### PR DESCRIPTION
**Pull request by zhaomin1423**
### _Why are the changes needed?_

The default value of kyuubi.engine.event.loggers is SPARK, but it can't be used outside the spark engine, so filter it.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala [hunk 1]
```diff
@@ -66,7 +65,6 @@ object HiveSQLEngine extends Logging {
   var currentEngine: Option[HiveSQLEngine] = None
   val hiveConf = new HiveConf()
   val kyuubiConf = new KyuubiConf()
-  kyuubiConf.set(ENGINE_EVENT_LOGGERS.key, "JSON")
```
Comment (ulysses-you) [inline externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala:69]: this is a behavior change.. the default value is `SPARK` and you just filter that

Comment (ulysses-you) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala:69]: one option maybe, we can add engine event loggers config for each engine.
e.g.
```
kyuubi.engine.spark.event.loggers
kyuubi.engine.jdbc.event.loggers
...
```

Comment (ulysses-you) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala:69]: cc @yaooqinn

Comment (zhaomin1423) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala:69]: > one option maybe, we can add engine event loggers config for each engine.
> e.g.
> ```
> kyuubi.engine.spark.event.loggers
> kyuubi.engine.jdbc.event.loggers
> ...
> ```

good idea

Comment (zhaomin1423) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala:69]: add config for each engine. https://github.com/apache/incubator-kyuubi/pull/3309#issue-1346621427

**Pull request closed without merging**